packages/esm-appointments-app/src/metrics/metrics-header.component.tsx

```diff
@@ -34,7 +36,7 @@ const MetricsHeader: React.FC = () => {
         <Button
           kind="tertiary"
           renderIcon={Calendar}
-          onClick={() => navigate({ to: `${spaHomePage}/appointments/calendar` })}>
+          onClick={() => navigate({ to: `${spaHomePage}/appointments/calendar/${selectedDate}` })}>
```

Comment:
Pass in the currently selected date when navigating to the calendar so that it opens to the correct month.